types/react-dom/experimental.d.ts

```diff
@@ -32,4 +32,21 @@ import ReactDOM = require('./next');
 
 export {};
 
-declare module '.' {}
+declare module '.' {
```

Comment:
What is this trick doing? Extensionless module specifier in ESM?

Reply:
It's referencing the current folder which then falls back to `index`. Could just write `react-dom` here but I wanted to be like the cool kids, write code that I can simply c&p into other modules (React has the same pattern) thus making my code DRY 😎